source/data-formats/serialization.txt

```diff
@@ -227,6 +227,24 @@ encode defaults:
    :end-before: end-codec
    :dedent:
 
+.. _kotlin-sync-serialization-snake-case-eg:
+
+Enforcing Snake Case Naming Example
```

Comment:
I: use present tense imperative, also consider dropping example as this isn't a full example.
```suggestion
Implement Snake Case Naming Strategy
```